plugins/outputs/postgresql/README.md

```diff
@@ -1,36 +1,44 @@
 # PostgreSQL Output Plugin
 
-This output plugin writes all metrics to PostgreSQL.
+This output plugin writes all metrics to PostgreSQL. 
+The plugin manages the schema automatically updating missing columns, and checking if existing ones are of the proper type. 
 
 ### Configuration:
 
 ```toml
 # Send metrics to postgres
 [[outputs.postgresql]]
-  address = "host=localhost user=postgres sslmode=verify-full"
-
-  ## Store tags as foreign keys in the metrics table. Default is false.
-  # tags_as_foreignkeys = false
-
-  ## Template to use for generating tables
-  ## Available Variables:
-  ##   {TABLE} - tablename as identifier
-  ##   {TABLELITERAL} - tablename as string literal
-  ##   {COLUMNS} - column definitions
-  ##   {KEY_COLUMNS} - comma-separated list of key columns (time + tags)
-
-  ## Default template
-  # table_template = "CREATE TABLE IF NOT EXISTS {TABLE}({COLUMNS})"
-  ## Example for timescaledb
-  # table_template = "CREATE TABLE IF NOT EXISTS {TABLE}({COLUMNS}); SELECT create_hypertable({TABLELITERAL},'time',chunk_time_interval := '1 week'::interval, if_not_exists := true);"
-
-  ## Schema to create the tables into
-  # schema = "public"
-
-  ## Use jsonb datatype for tags. Default is true.
-  # tags_as_jsonb = false
-
-  ## Use jsonb datatype for fields. Default is true.
-  # fields_as_jsonb = false
+    address = "host=localhost user=postgres sslmode=verify-full"
```

Comment:
1) Maybe you want to rename this to "connection" since this is not only the address?
2) I tried to set "sslcompression=1" but it seems like this keyword is unknown. Had no time to debug, should this work?